config/database.yml

```diff
@@ -1,8 +1,14 @@
+# If you want to change this file, please keep the changes in your working
+# copy by using
+#
+#     git update-index --skip-worktree config/database.yml
```

Comment:
> people don't read
Adding a comment in the code with a link to the documentation could help

> people might fork the repo, and not the wiki
I agree, but we're just trying to be nice adding documentation on how to customize the default setup